core/models/src/node/operations.rs

```diff
@@ -40,26 +40,25 @@ impl DepositOp {
         data
     }
 
-    pub fn from_bytes(bytes: &[u8]) -> Option<Self> {
+    pub fn from_public_data(bytes: &[u8]) -> Option<Self> {
```

Comment:
I was talking about using Result/failure here. 
This way you could know why transform from pubdata failed.

`.ok()?` and `return None` is non informative.

Reply:
Done just a few moments after you've written this